src/main/java/com/google/cloud/tools/appengine/cloudsdk/CloudSdk.java

```diff
@@ -79,22 +79,26 @@
   private final ProcessRunner processRunner;
   private final String appCommandMetricsEnvironment;
   private final String appCommandMetricsEnvironmentVersion;
-  @Nullable
   private final File appCommandCredentialFile;
   private final String appCommandOutputFormat;
+  private final String appCommandShowStructuredLogs;
   private final WaitingProcessOutputLineListener runDevAppServerWaitListener;
 
-  private CloudSdk(Path sdkPath, Path javaHomePath, String appCommandMetricsEnvironment,
-                   String appCommandMetricsEnvironmentVersion,
+  private CloudSdk(Path sdkPath, Path javaHomePath,
+                   @Nullable String appCommandMetricsEnvironment,
+                   @Nullable String appCommandMetricsEnvironmentVersion,
                    @Nullable File appCommandCredentialFile,
-                   String appCommandOutputFormat, ProcessRunner processRunner,
+                   @Nullable String appCommandOutputFormat,
+                   @Nullable String appCommandShowStructuredLogs,
+                   ProcessRunner processRunner,
```

Comment:
Is this Nullable? I don't see a null check in the constructor. Ditto sdkPath and javaHomePath.